modules/ROOT/pages/as2/AS2-connector-examples-6-0.adoc

```diff
@@ -0,0 +1,90 @@
+= AS2 - Examples - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+== Example: Configure a Send AS2 Message
+
+. Place the connector in your flow as applicable.
+. To send an AS2 message with request for MDN in your Mule application, create a global `AS2 Connector Send config` element for use by the AS2 send operations.
+The `AS2 Connector Send config`  provides the following global configuration parameters:
++
+[%header%autowidth.spread]
+|===
+|Key| Description
+|Self config - AS2 Partner Name| Sender’s AS2 system identifier. Used for AS2-From and AS2-To headers.
+|Self config - x509 alias| Sender’s certificate and/or private key alias in the keystore. Required if using signing or encryption.
+|Self config - Email| Sender partner’s email.
+|Partner config - AS2 ID| Receiver’s AS2 system identifier. Used for AS2-From and AS2-To headers.
+|Partner config - x509 alias| Receiver’s Certificate and/or private key alias in the keystore. Required if using signing or encryption.
+|Partner config - Email| Receiver partner’s email.
+|Requester config - Subject| Free-form text for MIME subject header. Can include references to self or partner `simpleName` values along with the message `fileName`, for example, sending the `file example.txt` to partner via AS2.
+|Requester config - MDN MIC Signature Algorithm| The algorithm with which MIC is calculated for the MDN signature. The message is `UNSIGNED` by default.
+|Requester config - Request a receipt| Option to request a receipt from the partner. `SIGNED_OPTIONAL` by default.
+|Requester config - Request delivery URL| Asynchronous receipt delivery URL that points to the MDN listener endpoint.
+|Requester config - Encryption Algorithm| The encryption algorithm for encrypting the message. `UNENCRYPTED` by default.
+|Requester config - MIC Signature Algorithm| The algorithm with which MIC is calculated for the message signature.  The message is `UNSIGNED` by default.
```

Comment:
There is a missing field for the MDN MIC Signature Algorithm, this has the following summary:
Request partner to respond with a signed MDN using this algorithm for calculating MIC.

Reply:
Add to description.

Reply:
`|Requester config - MDN MIC Signature Algorithm| Request partner to respond with a signed MDN using this algorithm for calculating MIC.`